app/scripts/views/generator-view.js

```diff
@@ -103,17 +123,22 @@ const GeneratorView = Backbone.View.extend({
 
     generate: function() {
         this.password = PasswordGenerator.generate(this.gen);
-        this.resultEl.text(this.password);
+        this.showPassword();
         const isLong = this.password.length > 32;
         this.resultEl.toggleClass('gen__result--long-pass', isLong);
     },
 
+    hideChange: function(e) {
+        this.hide = e.target.checked;
+        AppSettingsModel.instance.unset('generatorHide', { silent: true });
+        AppSettingsModel.instance.set('generatorHide', this.hide);
+        const label = this.$el.find('#gen__check-hide + label');
+        Tip.createTip(label[0], {title: this.hide ? Locale.genShowPass : Locale.genHidePass});
+        this.showPassword();
+    },
+
     btnOkClick: function() {
-        const selection = window.getSelection();
-        const range = document.createRange();
-        range.selectNodeContents(this.resultEl[0]);
-        selection.removeAllRanges();
-        selection.addRange(range);
+        CopyPaste.createHiddenInput(this.password);
```

Comment:
Please also check `!CopyPaste.simpleCopy` before.

Reply:
I think we should move the simpleCopy check and createHiddenInput call into the CopyPaste.copy function then.

Reply:
Let me know if 4892e7a suitably addresses this.

Reply:
No, please check `!CopyPaste.simpleCopy` here, modifying copy-paste will break other stuff